pl-csb-model/train/model_finetuner.py

```diff
@@ -24,13 +24,13 @@ def __cleanup(self) -> None:
 
     def __get_random_language_pairs(self, batch_size, langs, data):
         try:
-            (l1, long1), (l2, long2) = random.sample(langs, 2)
+            l1, l2 = random.sample(langs, 2)
```

Comment:
We've decided that the columns will use the same names as language names in the model instead of "pl" and "csb" which means we can just toss those away and use the other variable instead. I might rename it to lang1 and lang2 to make it more clear but its not that important.